packages/cli/src/Server.ts

```diff
@@ -479,6 +482,19 @@ class Server extends AbstractServer {
 			this.app.use(`/${this.restEndpoint}/ldap`, ldapController);
 		}
 
+		// ----------------------------------------
+		// SAML
+		// ----------------------------------------
+
+		// public SAML endpoints
+		this.app.use(`/${this.restEndpoint}/sso`, samlControllerPublic);
+
+		// licensed SAML endpoints
+		this.app.use(samlEnabledMiddleware);
```

Comment:
Won't adding the middleware here block all requests to the n8n api? Or would it only affect the SSO related endpoints?